# This expresses the nervous states broken into the emotions and behaviors assessment parts.
Emotions emotions
Behaviors behaviors
